cps/src/main/java/org/jenkinsci/plugins/workflow/cps/EnvActionImpl.java

```diff
@@ -108,4 +109,23 @@
         owner = r;
     }
 
+    @Extension public static class Binder extends Singleton {
+        @Override public String getName() {
+            return "env";
+        }
+        @Override public Object getProperty(CpsScript script) throws Exception {
+            Run<?,?> run = script.$build();
+            if (run != null) {
+                EnvActionImpl action = run.getAction(EnvActionImpl.class);
+                if (action == null) {
+                    action = new EnvActionImpl();
+                    run.addAction(action);
+                }
+                return action;
```

Comment:
Not relevant to this PR really but, does `Run` not have a `getEnv` method wrapping access to the actions etc? I know it's API could explode if you added everything to it, but this one seems fundamental to me.